docs/configuration/translation-strings.md

```diff
@@ -23,29 +11,26 @@ You can also translate [Collection Content]() and [Field Names]().
 </video>
 
 When configuring Fields as well as Presets and Bookmarks, you may wish to add additional notes or descriptions to
-clarify their purpose or context.
+clarify their purpose or context. To create and assign a Translation String, follow these steps.
 
-**Create and Manage Translation Strings**\
+**Create and Manage Translation Strings**
 
 1. Navigate to **Settings > Translation Strings**.
 2. Click <span mi btn>add</span> in the page header and a side menu will appear.
-3. Add a key and click **"Create New"**. Another side menu will appear.
+3. Add a key and click **"Create New"** and another side menu will appear.
 4. Select the language and add the corresponding translation.
 5. Click <span mi btn>check</span> to confirm and add the translation.
 6. Repeat steps 3-5 as desired.
 7. Optional:
    - Click and drag <span mi icon>drag_handle</span> to reposition a translation.
    - Click <span mi icon>close</span> to delete a translation.
 8. Click <span mi btn>check</span> to save the translation(s).
-9. Optional: Delete Translation String\
-   Click the translation string to open the side menu, click <span mi btn dngr>delete</span>, then click **Delete**.
 
-Now you can add it to Translation String Inputs in the **Settings Module**, noted with <span mi icon>translate</span> on
-them.
+In the Settings Module, you can assign the Translation String to any inputs with a <span mi icon>translate</span> icon.
 
-**Assign a Translation String**\
+**Assign a Translation String**
 
-1. Navigate to the desired Translation String Input within Settings.\
+1. Navigate to the desired Translation String Input within Settings.
```

Comment:
This step makes it seem like you're going to the Translation Strings page again. Let's be clearer in saying something like "Navigate to the desired input within Settings that you want to translate...". A list of examples would help, and of course any video/image.